how-to-pitch-post-supply/readme.md

```diff
@@ -0,0 +1,49 @@
+> Goal of this guide: Help you make it clear what Post.supply is and what it offers
+
+# Your awesomeness is enough. You don't have to be a selling super hero
+<img src="http://66.media.tumblr.com/da6ec2c2cb443e8e7d1ade9eb220c86d/tumblr_nvls4ybWUo1sxfvy5o1_400.gif" width="500px">
+
+First off it is really important that we should never try to sell our product.
+
+We offer a solution to a problem. As long as we are able to explain how we solve the problem, those that have the problem will see the value in what we do.
+
+The product should sell itself, as long as people know what the product is. If not, we need to redefine the product, not try to sell harder.
+
+## The basics of what we do and why we do it.
+
+
+### Problem
+
+Social media is mostly used to influence the way people perceive the sender, creating large amounts of aggressive and pushy content that has no real value for the receiver. This is a waist of resources and drowns out the content that is actually valuable for the receiver.
+
+### Solution
+
+We are convinced that you can have effective communication by using it as a tool to achieve the goals of your company.
+
+The best way to do this is by enabling your audience to be part of the solution to the problem your company is trying to solve.
+
+This way you show the importance of what you do, you validate your expertise, you are valuable to your audience and you get closer to achieving the goals of your company.
+
+### Our product
+
+We teach you how to do this trough our website. If you do not have the time to do it yourself, we can do it for you trough our application.
+
+**Our website** is the place where we publicly store all the lessons we have learned about reaching goals trough good content. Our website is always the first place where we look when researching for new articles. By making it public we hope to enable others to do this as well.
```

Comment:
trough-->through

"making it public"-->making our research public (it is a little ambiguous; seems to refer to our website😝)